Tests/docker_native_image_config.json

```diff
@@ -63,56 +63,42 @@
       }
    },
    "ignored_content_items":[
-      {
-         "id":"ParseEmailFilesV2",
-         "reason":"Issue: CIAC-5711 - Failed unit-test: test_eml_contains_empty_htm_not_containing_file_data.",
-         "ignored_native_images":[
-            "native:8.1"
-         ]
-      },
       {
          "id":"UnzipFile",
          "reason":"Issue: CIAC-5246 - Failed unit-test: test_unrar_no_password.",
          "ignored_native_images":[
-            "native:8.1",
+            "native:8.2",
             "native:dev"
          ]
       },
       {
          "id":"DockerHardeningCheck",
          "reason":"Issue: CIAC-5248 - Failed unit-test: get_default_gateway (ip command does not exist).",
          "ignored_native_images":[
-            "native:8.1",
+            "native:8.2",
             "native:dev"
          ]
       },
       {
          "id":"FetchIndicatorsFromFile",
          "reason":"issue: CIAC-5243 - ElementTree object (xml) in python3.9 has getiterator attribute while in python 3.10 this attribute does not exist - causing unit tests failures.",
          "ignored_native_images":[
-            "native:8.1",
+            "native:8.2",
             "native:dev"
          ]
       },
       {
          "id":"Traps",
          "reason":"Integration is Deprecated.",
          "ignored_native_images": [
-            "native:8.1",
+            "native:8.2",
             "native:dev"
          ]
-      },
-      {
-         "id":"Rasterize",
-         "reason":"Issue: CRTX-72338.",
-         "ignored_native_images":[
-            "native:8.1"
-         ]
```

Comment:
CRTX-72338 is fixed?

Reply:
if not then how was native:dev not in the list before?

Reply:
since it passes the UTs, the problem is on platform side, we still want to make sure it works with `native:dev`, in case this issue hasn't been resolved in platform side, we should still ignore it in the `native:ga` but not in `native:dev`